ironfish/src/fileStores/config.ts

```diff
@@ -245,6 +244,16 @@ export type ConfigOptions = {
   feeEstimatorPercentileLow: number
   feeEstimatorPercentileMedium: number
   feeEstimatorPercentileHigh: number
+
+  /**
+   * Network ID of an official Iron Fish network
+   */
+  networkId: number
+
+  /**
+   * JSON file containing the network definition of a custom network
```

Comment:
Maybe "Path to a JSON file", not an important change though

Reply:
Changed